### PR TITLE
Feature/timer

### PR DESCRIPTION
**Pull request by akmal-threepointsix**
- I added timer feature
- It is completely working, i.e. you can set timer, pause/resume, reset, or simply wait till it ends
- When it ends, it uses notification system (`NotificationClient` and `OswServiceTaskNotifier`) to notify the user

Some screenshots of the timer:

1. Initial screen
image
2. Set timer screen
image
3. Running timer screen
image

### CODE REVIEW

File: include/locales/en-US.h [hunk 1]
```diff
@@ -12,6 +12,12 @@
 #define LANG_CANCEL "Cancel"
 #define LANG_SAVE "Save"
 
+// App: Timer
+#define LANG_TIMER "Timer"
+#define LANG_START "Start"
+#define LANG_PAUSE "Pause"
+#define LANG_RESUME "Resume"
+
```
Comment (cutieskye) [inline include/locales/en-US.h:20]: We need to devise a way for an application to fall back to the English language if it has not been localized yet.

**Review by cutieskye — CHANGES_REQUESTED** [review]

File: include/locales/en-US.h (hunk 1 above)
Comment (akmal-threepointsix) [reply include/locales/en-US.h:20]: I think your idea is good, but it should be opened as a separate issue and implemented in a separate branch.
This PR only implements the Timer feature.

Comment (akmal-threepointsix) [reply include/locales/en-US.h]: Fixed the error

File: include/apps/clock/OswAppTimer.h [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    // Helpers
+    void timestampToSec();
+    void resetTimer();
+
+    // Draw on screen
+    void drawNumber(const int number, const int index);
+    void drawTime(const int totalSeconds);
+
+    OswAppSwitcher *clockAppSwitcher{};
+    NotifierClient notifierClient{"com.akmal-threepointsix.application.timer"};
```
Comment (simonmicro) [inline include/apps/clock/OswAppTimer.h:47]: IIRC you plan to merge this into OSW itself -> maybe use `org.open-smartwatch.osw.timer` instead?

File: include/config_defaults.h [hunk 3]
```diff
@@ -217,15 +217,15 @@
 #ifndef TOOL_CALCULATOR
 #define TOOL_CALCULATOR 1
 #endif
-#ifndef TOOL_STOPWATCH
-#define TOOL_STOPWATCH 1
-#endif
```
Comment (simonmicro) [inline include/config_defaults.h:222]: Why was this removed? Does your app replace the timer? No. Did you replace the flag? No.

Also you missed one in the `CMakeLists.txt` for the emulator.

Maybe you could introduce a new flag to make the whole "clock-apps" section optional.

**Review by simonmicro — CHANGES_REQUESTED** [review]

**Review by simonmicro — APPROVED** [review]
